src/client/BrowseLobbyModal.ts

```
@@ -0,0 +1,543 @@
+      console.warn("BrowseLobbyModal: Failed to get wallet address:", error);
+      return undefined;
+    }
+  }
+
+  render() {
+    return html`
+      <o-modal title="Browse Public Lobbies">
+        <!-- Header (always visible) -->
+        <div class="browse-header">
+          <div class="lobby-count">
+            ${this.lobbies.length} public ${this.lobbies.length === 1 ? 'lobby' : 'lobbies'}
```

ESLint

[error] Strings must use doublequote.

(@stylistic/quotes)